FirstProject/css/main.css

```diff
@@ -0,0 +1,149 @@
+
+/*Header section*/
+header {
+    background-image: url(../img/feedback-banner.jpg);
+    width: 100%;
+}
+
+.left_image {
```

Comment:
If you are referring to container-fluid class, that is a particular class(Bootstrap). I have to leave it like that..